core/io/image_loader.h

```diff
@@ -77,11 +78,13 @@ class ImageFormatLoaderExtension : public ImageFormatLoader {
 public:
 	virtual Error load_image(Ref<Image> p_image, Ref<FileAccess> p_fileaccess, BitField<ImageFormatLoader::LoaderFlags> p_flags = FLAG_NONE, float p_scale = 1.0) override;
 	virtual void get_recognized_extensions(List<String> *p_extensions) const override;
+	virtual bool should_import(const String &p_resource_type) const override;
```

Comment:
`virtual bool should_import(const String &p_resource_type) const override;` I think this was added so that Godot Engine can ignore this format loader?

Reply:
Yeah with #100538, `should_import` enables format loaders to designate when they would and would not be ignored by the import process based on the resource type.

Reply:
This feels wrong on a gut feeling; I've been ignoring this problem in my other pull request. https://github.com/godotengine/godot/pull/101994